{{ cookiecutter.project_slug }}/src/helpers/health_check_middleware.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+
+
+class HealthCheckMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+    
+    def __call__(self, request):
+        if request.path.replace("/", "") == "health_check":
+            request.META["HTTP_HOST"] = "localhost"
```

Comment:
I've just noticed this. I don't think this is a good approach, as some may use either `0.0.0.0` or even `127.0.0.1`. Maybe check the current running host before? Dunno.

Reply:
The default value for `ALLOWED_HOSTS` when it is empty is `['localhost', '127.0.0.1', '[::1]']`, so if we set the `HTTP_HOST` value with `localhost` it always will be bypassed in allowed hosts validation. But if an `ALLOWED_HOSTS` value is given, it will get from it.